change/@fluentui-react-positioning-304049a8-2b2a-4fbf-8a94-2cef2af9ab0d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29928&commitId=f60104b63ee0f29020a03d85685518bdc702058a&env=prod&clientType=FLUENTUI)


  <details><summary><b>InfoLabel</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|InfoLabel.sizes -- open.rest.chromium.png|4840| Changed|

</details>


  <details><summary><b>Positioning</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.auto size with async content.floating element is within the boundary.chromium.png|2911| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
https://github.com/floating-ui/floating-ui/pull/2382 causing the regressions